imqsauth/http.go

```diff
@@ -770,3 +847,72 @@ func httpHandlerGetGroups(central *ImqsCentral, w http.ResponseWriter, r *httpRe
 		httpSendGroupsJson(w, groups)
 	}
 }
+
+func (central *ImqsCentral) handleWebSocket(ws *websocket.Conn) {
+	closeConnection := true
+	if token, err := authaus.HttpHandlerPrelude(&central.Config.Authaus.HTTP, central.Central, ws.Request()); err == nil {
+		if permList, errDecodePerms := authaus.PermitResolveToList(token.Permit.Roles, central.Central.GetRoleGroupDB()); errDecodePerms == nil {
+			closeConnection = !permList.Has(PermEnabled)
+		}
+	}
+
+	if closeConnection {
+		ws.Close()
+		return
+	}
+
+	id := central.addSubscriber(ws)
+
+	central.Central.Log.Infof("Added auth web socket client %d", id)
```

Comment:
I think it would be useful to include the identity name in this log message. Also.. just use `%v` everywhere instead of `%d` etc, unless you REALLY need more specific formatting. It just makes for less brittle code, eg if the type of `id` changes from numeric to string.